diff -rupN a/src/file.cpp b/src/file.cpp
--- a/src/file.cpp	2015-01-06 16:59:36.321287262 -0500
+++ b/src/file.cpp	2015-01-06 16:58:41.000000000 -0500
@@ -307,7 +307,12 @@ File::FileType getFileType(const QString
         }
         else if (left == QLatin1String("text")
                  || mime == QLatin1String("application/x-zerosize")
-                 || mime == QLatin1String("application/x-shellscript")) {
+                 || mime == QLatin1String("application/x-shellscript")
+                 || mime == QLatin1String("application/x-perl")
+                 || mime == QLatin1String("application/x-desktop")
+                 || mime == QLatin1String("application/xml")
+                 || mime == QLatin1String("application/javascript")
+                 || mime == QLatin1String("application/json")) {
             type = File::Txt;
         }
         else if(mime.mid(delimiter + 1) == QLatin1String("directory")) {
